tests/PySys/tedge_connect_test_positive/run.py

```diff
@@ -20,16 +20,26 @@
 class TedgeConnectTestPositive(EnvironmentC8y):
     def execute(self):
         super().execute()
+        self.systemctl="/usr/bin/systemctl"
         self.log.info("Execute `tedge connect c8y --test`")
         self.startProcess(
             command=self.sudo,
             arguments=[self.tedge, "connect", "c8y", "--test"],
             stdouterr="tedge_connect_c8y_test_positive",
         )
 
+        self.startProcess(
+            command=self.sudo,
+            arguments=[self.systemctl, "status", "tedge-agent.service"],
+            stdouterr="tedge_agent_status",
+        )
+
     def validate(self):
         super().validate()
         self.log.info("Validate")
         self.assertGrep(
             "tedge_connect_c8y_test_positive.out", "connection check is successful.", contains=True
         )
+        self.assertGrep(
+            "tedge_connect_c8y_test_positive.out", "connection check is successful.", contains=True
+        )
```

Comment:
Sounds unfinished.

* What is tested here?

Reply:
Added new tests, reverted these changes